code/crates/test/tests/it/validator_set.rs

```diff
@@ -0,0 +1,26 @@
+use std::time::Duration;
+
+use informalsystems_malachitebft_test::middleware::RotateValidators;
+
+use crate::TestBuilder;
+
+#[tokio::test]
+async fn rotate_validator_set() {
+    const HEIGHT: u64 = 3;
```

Comment:
Let's increase the height a bit just to make sure:

```suggestion
    const HEIGHT: u64 = 9;
```